libcaliptra/test/hwmod_test/hwmod_if_impl.c

```diff
@@ -0,0 +1,82 @@
+//Licensed under the Apache-2.0 license
+
+#include <stdint.h>
+#include <stddef.h>
+#include <stdbool.h>
+
+#include "caliptra_model.h"
+#include "caliptra_api.h"
+
+#define CALIPTRA_STATUS_OK 0
+
+// Implementation specifics
+
+struct caliptra_model_init_params init_params;
+struct caliptra_fuses fuses = {0};
+
+static bool caliptra_model_init_complete = false;
```

Comment:
where is this used ?

Reply:
I think that's a leftover from when I had a more complicated init setup, I will remove it.